tools/cloudera/build_hue_cloudera.sh

```diff
@@ -51,8 +51,10 @@ function install_prerequisite() {
   export SQLITE3_PATH=${SQLITE3_PATH:-"$TOOLS_HOME/sqlite/sqlite3"}
   if [[ $1 == "redhat9" || $1 == "redhat9_ppc" ]]; then
     check_python39_path
-  elif [[ $1 == "ubuntu22" || $1 == "sles15" ]]; then
+  elif [[ $1 == "ubuntu22" ]]; then
     check_python310_path
+  elif [[ $1 == "sles15" ]]; then
```

Comment:
So we don't support sles15 with python3.10 anymore? and it will only work with python3.11?

Reply:
@athithyaaselvam Yes, we will now support py3.11 on sles15. All sles15 builds (internal) will be tied to py3.11.